### PR TITLE
improve examples

### PR DESCRIPTION
**Pull request by StarpTech**
This PR will improve the schema examples to underline where they belong.

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [X] documentation is changed or added
- [X] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Validation-and-Serialization.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ const schema = {
     required: ['x-foo']
   }
 }
+
+fastify.post('/the/url', { schema })
```
Comment (delvedor) [inline docs/Validation-and-Serialization.md]: `fastify.post('/the/url', { schema }, handler)` or `fastify.post('/the/url', { schema }, () => {})`

Comment (StarpTech) [reply docs/Validation-and-Serialization.md]: Done

**Comment by StarpTech** [comment]
What's wrong with travis?

**Comment by StarpTech** [comment]
@mcollina @delvedor could you merge it? Travis is hanging.

**Comment by jsumners** [comment]
Did this solve #693?

**Comment by mcollina** [comment]
I think that one is a bug on fast-json-stringify.

**Pull request merged**